docs/weave/welcome.md

```diff
@@ -0,0 +1,120 @@
+| [Multisig](https://github.com/iov-one/weave/tree/master/x/multisig)              | Supports first-class multiple signature contracts, and allow modification of membership                                                                              |
+| [AtomicSwap](https://github.com/iov-one/weave/tree/master/x/aswap)               | Supports HTLC for cross-chain atomic swaps, according to the [IOV Atomic Swap Spec](https://github.com/iov-one/iov-core/blob/master/docs/atomic-swap-protocol-v1.md) |
+| [Escrow](https://github.com/iov-one/weave/tree/master/x/escrow)                  | The arbiter can safely hold tokens, or use with timeouts to release on vesting schedule                                                                              |
+| [Governance](https://github.com/iov-one/weave/tree/master/x/gov)                 | Hold on-chain elections for text proposals, or directly modify application parameters                                                                                |
+| [PaymentChannels](https://github.com/iov-one/weave/tree/master/x/paychan)        | Unidirectional payment channels, combine micro-payments with one on-chain settlement                                                                                 |
+| [Distribution](https://github.com/iov-one/weave/tree/master/x/distribution)      | Allows the safe distribution of income among multiple participants using configurations. This can be used to distribute fee income.                                  |
+| [Batch](https://github.com/iov-one/weave/tree/master/x/batch)                    | Used for combining multiple transactions into one atomic operation. A powerful example is in creating single-chain swaps.                                            |
+| [Validators](https://github.com/iov-one/weave/tree/master/x/batch)               | Used in a PoA context to update the validator set using either multisig or the on-chain elections module                                                             |
+| [Username](https://github.com/iov-one/weave/tree/master/cmd/bnsd/x/nft/username) | Example nft used by bnsd. Maps usernames to multiple chain addresses, including reverse lookups                                                                      |
+| [MessageFee](https://github.com/iov-one/weave/tree/master/x/msgfee)              | Validator-subjective minimum fee module, designed as an anti-spam measure.                                                                                           |
+| [Utils](https://github.com/iov-one/weave/tree/master/x/utils)                    | A range of utility functions such as KeyTagger which is designed to enable subscriptions to database.                                                                |
+
+### In Progress
+
+Light client proofs, custom token issuance and support for IBC (Inter Blockchain Communication) are currently being designed.
```

Comment:
To keep this current it may be better to point readers to the IOV blog which will announce new features etc. It also means you don't need to keep this updated!